src/dialog-editor/components/modal/modalComponent.ts

```diff
@@ -249,7 +249,7 @@ class ModalController {
     _.forEach(this.categories.resources, function (name) {
       if(name['id'] === item) {
         vm.modalData.options.category_description = name['description'];
-        vm.modalData.options.category_name = _.snakeCase(name['description']);
+        vm.modalData.options.category_name = name['name'];
```

Comment:
@eclarizio - There had to be a reason we decided to put `_.snakeCase` in there before right?  https://github.com/ManageIQ/ui-components/commit/8d68195c959da670a513baf5386a57b4aa91ceb3#diff-50f2063e2c8663d66ad3f5dd8131f2fdR21

I thought snakeCase was needed for a lookup on the backend ( at least when that PR was built. )

Reply:
Nevermind - I see that id is coming in from the ui-classic PR you referenced.

Reply:
@syncrou Yes, we do use name on the backend for a lookup, but name and description are not required to be the same.  For example the name for `Cost Center` is `cc` and the name for `Workload` is `function`.  So using snakeCase here returns a name that fails the backend lookup.